multimedia/tvheadend/patches/060-busybox-tar.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/config.c
++++ b/src/config.c
+@@ -1481,7 +1481,7 @@ dobackup(const char *oldver)
```

Comment:
I don't like patches, where there are no details like who is author, why it is necessary, etc. If you are using Tvheadend, then you should use proper tar, but if you want to use busybox tar, then please try to discuss it with upstream to include the solution there.

Reply:
I can remove it completely. It won't give any error unless someone changes the startup script.

Reply:
There is also another possibility add tar as dependency to tvheadend until someone changes the startup script.